app/views/repository_columns/_manage_column_modal.html.erb

```diff
@@ -17,47 +17,60 @@
-              <%= f.radio_button :data_type,
-                                 'RepositoryListValue'.freeze,
-                                 label: t('libraries.manange_modal_column.labels.dropdown'),
-                                 inline: true,
-                                 checked: checked?(@repository_column, 'RepositoryListValue'.freeze),
-                                 disabled: disabled?(@repository_column, 'RepositoryListValue'.freeze) %>
-            </span>
-          <% end %>
-          <input class="form-control"
-                 data-role="tagsinput"
-                 style="display: none"
-                 data-value="<%= @repository_column.repository_list_items.pluck(:data) %>"
-                 />
-          <br />
-          <span style="display: none" data-role="tagsimput-label"><%=t 'libraries.manange_modal_column.dropdown_item_descirption' %></span>
-        <% end %>
+        <div class="form-horizontal" id="new_repository_column">
+          <div class="form-group">
+            <label class="control-label col-sm-3" for="repository_column_name">
+              <%= t('libraries.manange_modal_column.name') %>
+            </label>
+            <div class="col-sm-9">
+              <input placeholder="<%= t('libraries.manange_modal_column.name_placeholder') %>"
+                     class="form-control"
+                     type="text"
+                     name="repository_column[name]"
+                     id="repository_column_name">
+            </div>
+          </div>
+          <div class="form-group">
+            <label class="control-label col-sm-3" for="repository_column_data_type">
+              <%= t('libraries.manange_modal_column.colum_type') %>
+            </label>
+            <div class="col-sm-9">
+              <select class="form-control" name="repository_column[data_type]" id="repository_column_data_type">
+                <option data-create-url="<%= repository_create_text_path(@repository) %>" value="RepositoryTextValue">
+                  <%= t('libraries.manange_modal_column.select.repository_text_value') %>
+                </option>
+                <option data-create-url="<%= repository_create_date_path(@repository) %>" value="RepositoryDateValue">
+                  <%= t('libraries.manange_modal_column.select.repository_date_value') %>
+                </option>
+                <option data-create-url="<%= repository_create_list_path(@repository) %>" value="RepositoryListValue">
+                  <%= t('libraries.manange_modal_column.select.repository_list_value') %>
+                </option>
+                <option data-create-url="<%= repository_create_asset_path(@repository) %>" value="RepositoryAssetValue">
+                  <%= t('libraries.manange_modal_column.select.repository_asset_value') %>
+                </option>
+              </select>
+            </div>
+          </div>
```

Comment:
Clean all bootstrap elements (col-sm-9, form-group, form-control and etc). Make it more clean.